### ISSUE TITLE
Circuit Abbey Website down

### ISSUE THREAD

**Issue by riverbasin** [raphschwarz/Circuit-Abbey-G8-DIY-EDIT#1]
Hello raphschwarz.  It looks like the Circuit Abbey website is down - hence all of the supporting material for this module, including firmware, are not available.  Do you know of an alternate source for supporting documentation and firmware? Best, riverbasin

**Comment by raphschwarz** [comment]
Oups no I don’t !

I if doesn’t come back up I’ll ask around . I never actually had the panel made. So I never flashed the firmware on my build

**Comment by riverbasin** [comment]
Okay, good to know.

You must be aware that Pusherman productions is producing this panel and
PCB as a DIY project? Here’s the link:

https://pushermanproductions.com/product/g8-clock-divider-pcb-aluminium-panel/

Best, Byron

**Comment by raphschwarz** [comment]
Yes I know ! someone in pusherman groups must have it. If you find it don’t hesitate to send it to me I’ll host it with the board files so it is safely hosted for future generations ahaha

**Comment by riverbasin** [comment]
Sounds great. I have contacted Phil at Pusherman as well as posted on the
Pusherman Facebook page. We’ll see what comes of it. I will keep you posted.

Also I did some more searching, and I found out that Circuit Abby is having
some medical issues. Check out their Facebook page. They seem in good
spirits, but their webpage will be down for a while.

Best, Byron

**Comment by riverbasin** [comment]
All right.  Some smart person over at Pusherman Facebook page suggested the
obvious-in-hindsight web archive to go to a cached version of the Circuit
Abbey webpage.  The critical files are attached.

Best, Byron
